util.py

```diff
@@ -298,3 +299,58 @@ def buildCache(scriptDir, collectionDir, collection, logger):
     logger.log("gameplayPicCache: %i entities" % len(gameplayPicCache.keys()))
 
     return frontPicCache, titlePicCache, gameplayPicCache
+
```

Comment:
With the converter default beeing of not overwriting a game if it's found, I think it would be better to make this detection and handling a full separate functionnality in another PR, associated to a parameter.

Moving the game would likely break some of them too